modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -121,13 +124,15 @@ types:
     properties:
        a: string
        b: integer
-    examples: !include fragment.raml
+    examples: !include fragment.yaml
 ----
 
-* In the NamedExample fragment, ensure that each example consists of a name and a value, as in the fragment below that contains one example:
+??? Now that I've changed the example to be correct, the section title and intro do not match it.???
```

Comment:
I think it's ok, you're still documenting the issues/misconceptions with named examples. You could still rename that section to something a little more descriptive though, something like `Problems with RAML NamedExample fragment and examples in general`